### PR TITLE
UI-2352 Add margin to popup container

### PR DESCRIPTION
**Pull request by hanzge**
- Add `margin`attribute to popup data and popup info
- Add example to demo
- Add example to marker group demo

### CODE REVIEW

File: demo/px-map-marker-group-demo.html [hunk 1]
```diff
@@ -200,6 +200,7 @@
                     "popup-base": "data-popup",
                     "popup-title": "Cabrillo National Monument",
                     "popup-opened": "true",
+                    "popup-margin": "10px",
```
Comment (mallik-svmx) [inline demo/px-map-marker-group-demo.html]: can we keep it to Number type similar to width prop?

File: demo/px-map-popup-info-demo.html [hunk 2]
```diff
@@ -65,7 +65,7 @@
           attribution-prefix='<a href="http://leafletjs.com" title="A JS library for interactive maps">Leaflet</a> | &copy; <a href="http://osm.org/copyright">OpenStreetMap contributors</a>'>
           <px-map-tile-layer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"></px-map-tile-layer>
           <px-map-marker-static lat="42.3518414" lng="-71.0500149" type="info">
-            <px-map-popup-info min-width="200" title="{{props.title.value}}1233" description="{{props.description.value}}">
+            <px-map-popup-info margin="10px" min-width="200" title="{{props.title.value}}1233" description="{{props.description.value}}">
```
Comment (mallik-svmx) [inline demo/px-map-popup-info-demo.html]: (minor) can we remove 1233 test? possibly it was from earlier PR

File: demo/px-map-marker-group-demo.html (hunk 1 above)
Comment (hanzge) [reply demo/px-map-marker-group-demo.html]: I made it string in case they want to use percentage margin.

File: px-map-behavior-popup.es6.js [hunk 3]
```diff
@@ -41,6 +41,11 @@
         value: false
       },
 
+      margin: {
```
Comment (joshsylvester) [inline px-map-behavior-popup.es6.js]: needs a comment describing the new prop

Comment (hanzge) [reply px-map-behavior-popup.es6.js]: @joshsylvester Updated

**Comment by joshsylvester** [comment]
merging with approvals. @hanzge please draft a release & tag and release it!

**Pull request merged**